javascript/lang/en_GB.js

```diff
@@ -13,6 +13,7 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "UserForms.ADDING_OPTION": "Adding option",
     "UserForms.ADDING_RULE": "Adding rule",
     "UserForms.CONFIRM_DELETE_ALL_SUBMISSIONS": "All submissions will be permanently removed. Continue?",
+	"UserForms.ERROR_CONTAINER_HEADER": "Please correct the following errors and try again:",
```

Comment:
@robbieaverill we don't *need* to enforce updates to only those languages - we can just warn contributors that their other translations will be overwritten unless updated in transifex.